cmake/ProcessOptions.cmake

```
@@ -379,16 +379,12 @@ endfunction()
 function( NEST_PROCESS_WITH_PYTHON )
   # Find Python
   set( HAVE_PYTHON OFF PARENT_SCOPE )
-  if ( ${with-python} STREQUAL "ON" OR  ${with-python} STREQUAL "2" OR  ${with-python} STREQUAL "3" )
+  if ( ${with-python} STREQUAL "2" )
+    message( FATAL_ERROR "Python 2 is not supported anymore, please use Python 3." )
```

After the last update to cmakeLists.txt (this PR seems to be a moving target) 'with-python=3' will raise a cmake error. This error messge above could be confusing.
```suggestion
    message( FATAL_ERROR "Python 2 is not supported anymore, please use Python 3 by setting with-python=ON" )
```